cli/azd/internal/appdetect/appdetect_test.go

```diff
@@ -307,6 +322,11 @@ func TestDetect(t *testing.T) {
 					Language:      Java,
 					Path:          "java",
 					DetectionRule: "Inferred by presence of: pom.xml",
+					Options: map[string]interface{}{
+						JavaProjectOptionMavenPath:             filepath.Join(dir, "java"),
+						JavaProjectOptionPosixMavenWrapperPath: filepath.Join(dir, "java", "mvnw"),
+						JavaProjectOptionWinMavenWrapperPath:   filepath.Join(dir, "java", "mvnw.cmd"),
```

Comment:
Could you please help to improve this (mvn command related codes) after this PR merged: https://github.com/azure-javaee/azure-dev/pull/100

Reply:
Ok, will handle this. https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2331044

Reply:
Thank you. And maybe support multiple pom files in one folder in the new task. For example:
```
./project
./project/frontend-pom.xml
./project/backend-pom.xml
```

In this case, maybe we can change `JavaProjectOptionCurrentPomDir` to `JavaProjectOptionCurrentPomPath`

Reply:
I currently have no clear context for the PR https://github.com/azure-javaee/azure-dev/pull/100, will consider your mentioned case, thanks for your reminder.